src/linux/osd.c

```diff
@@ -71,9 +71,9 @@ ssize_t ofi_get_hugepage_size(void)
 	return val * 1024;
 }
 
-#ifdef HAVE_ETHTOOL
+#if HAVE_ETHTOOL == 1
```

Comment:
Minor style quibble: we have precedent elsewhere in the Libfabric code to treat such macros like booleans, and use syntax like this:

```c
#if FOO
// and
#if !FOO
```

(as opposed to `#if FOO == 1`)